src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -81,47 +81,18 @@ LoadFeeTrack::lowerLocalFee ()
 
 //------------------------------------------------------------------------------
 
-// NIKB TODO: Once we get C++17, we can replace lowestTerms
-//            with this:
-//
-// template <class T1, class T2,
-//     class = std::enable_if_t<
-//         std::is_integral_v<T1> &&
-//         std::is_integral_v<T2>>
-// >
-// void lowestTerms(T1& a,  T2& b)
-// {
-//     if (auto const gcd = std::gcd(a, b))
-//     {
-//         a /= gcd;
-//         b /= gcd;
-//     }
-// }
-
 template <class T1, class T2,
-    class = std::enable_if_t <
-        std::is_integral<T1>::value &&
-        std::is_unsigned<T1>::value &&
-        sizeof(T1) <= sizeof(std::uint64_t) >,
-    class = std::enable_if_t <
-        std::is_integral<T2>::value &&
-        std::is_unsigned<T2>::value &&
-        sizeof(T2) <= sizeof(std::uint64_t) >
+    class = std::enable_if_t<
+        std::is_integral_v<T1> &&
+        std::is_integral_v<T2>>
 >
 void lowestTerms(T1& a,  T2& b)
 {
-    if (a == 0 && b == 0)
-        return;
-
-    std::uint64_t x = a, y = b;
-    while (y != 0)
+    if (auto const gcd = std::gcd(a, b))
```

Comment:
Needs `#include <numeric>`.

Reply:
Fixed in b5a58480b `Add missing include for std::gcd`